lib/web_ui/dev/goldens_lock.yaml

```diff
@@ -1,2 +1,2 @@
 repository: https://github.com/flutter/goldens.git
-revision: f868ee6b6faef13469498e9b160b45fed9fdc567
+revision: a6211b5af9e5baa3b088440c42aad0914a4f1822
```

Comment:
Leave it at `f868ee6b6faef13469498e9b160b45fed9fdc567`. That's the newest commit.